Moda.Web/src/moda.web.reactclient/src/app/components/common/planning/create-risk-form.tsx

```diff
@@ -181,15 +156,13 @@ const CreateRiskForm = ({
           const loadData = async () => {
             setTeamOptions(await getTeamOptions())
             setEmployeeOptions(await getEmployeeOptions())
```

Comment:
Since employee options is used on both the create and edit forms, can it be made into a query as well?

Reply:
I was going to do the organization queries in the next PR if this all looks good.